docs/reference/index.rst

```diff
@@ -23,7 +23,8 @@ Persistence Images
     :toctree: stubs
     :nosignatures:
 
-    persim.PersImage
+    persim.PersistenceImager
```

Comment:
When I try to build the documentation, I get `AttributeError: module 'persim' has no attribute 'PersistenceImager'`. Looks like it might be because of this line.

Reply:
I am not able to reproduce this error.  I did find that I had to `python setup.py install` (and make sure the install really did overwrite) before running `make html` otherwise local changes were not showing up in the documentation.  This error suggests it is somehow still accessing the current version on images.py which does not have PersistenceImager defined?

After you fixed the issue with distances.ipynb, the related warnings in the documentation build have gone away.  I only get 3 warnings with `make html`, again not related to PersistenceImager:

```
\local\continuum\anaconda3\envs\persimenv\lib\site-packages\persim-0.1.4-py3.7.egg\persim\gromov_hausdorff.py:docstring of persim.gromov_hausdorff.gromov_hausdorff:: WARNING: Undefined substitution referenced: "V(G)|×|V(G)".

\local\continuum\anaconda3\envs\persimenv\lib\site-packages\persim-0.1.4-py3.7.egg\persim\gromov_hausdorff.py:docstring of persim.gromov_hausdorff.gromov_hausdorff:: WARNING: Undefined substitution referenced: "V(H)|×|V(H)".

checking consistency... \persim\docs\notebooks\Persistence barcode measure.ipynb: WARNING: document isn't included in any toctree
```

Reply:
:D If you're interested in looking into those issues, that'd be rad. no worries though. I'll take another stab making the html again.  🤔

Reply:
Happy to take a look!

Reply:
Fixed these issue with updates to the docstring in persim.gromov_hausdorff.gromov_hausdorff and added  Persistence barcode measure.ipynb to the index.rst under tutorials.